ovn-tester/ovn_workload.py

```diff
@@ -143,7 +157,8 @@ def enable_txns_db_logging(self):
 
     def get_connection_string(self, cluster_cfg, port):
         protocol = "ssl" if cluster_cfg.enable_ssl else "tcp"
-        ip = self.mgmt_ip
+        off = 3 * self.id if cluster_cfg.clustered_db else self.id
+        ip = self.mgmt_ip + off
         num_conns = 3 if cluster_cfg.clustered_db else 1
         conns = [f"{protocol}:{ip + idx}:{port}" for idx in range(num_conns)]
         return ",".join(conns)
```

Comment:
I think we can simplify the way connection strings are computed (and we can also get rid of `calculate_node_remotes()`) if we refactor a bit the Node -> CentralNode/WorkerNode hierarchy.  I'll work on a separate PR for that.